spec/requests/patreon_webhook_controller_spec.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'openssl'
+require 'json'
+require_relative '../spec_helper'
+
+RSpec.describe ::Patreon::PatreonWebhookController do
```

Comment:
That file does not look like it belongs here 😛

Reply:
😱

Reply:
Removed